bc_gym_planning_env/robot_models/robot_dimensions_examples.py

```diff
@@ -186,7 +186,6 @@ def footprint(dynamic_footprint_modifier=0.):
             [1303.15, -286.54],
             [1341.34, -118.37]
         ]) / 1000.
-        footprint[:, 1] *= 0.95
```

Comment:
This was not in the last version of the ENV given to me. Don't see why this need to be changed. Which version should we keep?